app/assets/javascripts/models/data/service_options.js.coffee

```diff
@@ -176,10 +176,14 @@ ns.ServiceOptions = do (ko, edsc = @edsc, KnockoutModel = @edsc.models.KnockoutM
       return true if @granuleAccessOptions().methods?.length == 0
       
       result = false
+      selected = false
       for m in @accessMethod()
+        if @granuleAccessOptions().methods?.length == 1
+          m.method(m.availableMethods[0].name)
         result = true if m.method()? && m.isValid()
         result = true if !m.loadForm()
-      result
+        selected = true if m.serialize().method != null
```

Comment:
Here you're using `m.serialize().method != null` but just above the previous code is using `m.method()?` -- do these two methods of checking for null differ?